docs/visualization/lupton_rgb.rst

```diff
@@ -0,0 +1,26 @@
+**********************************
+Creating color RGB images
+**********************************
+
+`Lupton et al. (2004)`_ describe an "optimal" algorithm for producing red-green-
+blue composite images from three separate high-dynamic range arrays. This method
+is implemented in `astropy.visualization.lupton_rgb` as a set of classes
+providing different scalings and a convenience wraper function. To generate a
+color PNG file with the default (arcsinh) scaling:
+
+>>> from astropy.visualization import lupton_rgb
```

Comment:
You can, but you can use the ``.. plot`` directive. Use the example from e.g. ``docs/visualization/wcsaxes/index.rst``

For the plotting, though, you need to give a full standalone script every time. Adding ``:include-source:``  will repeat the code in the docs, too, so you don't need to have it in there twice